src/Microsoft.AspNetCore.SignalR.Protocols.MessagePack/Protocol/MessagePackHubProtocol.cs

```diff
@@ -16,6 +16,9 @@
 
 namespace Microsoft.AspNetCore.SignalR.Protocol
 {
+    /// <summary>
+    /// A MessagePack protocol for communicating with SignalR hubs.
```

Comment:
Implements the SignalR Hub Protocol using MessagePack.

(Also, can we link to websites? Should we?)